internal/compiler/widgets/common/common.slint

```diff
@@ -11,7 +11,7 @@ export LineEditInner := Rectangle {
     property text <=> input.text;
     property placeholder-color <=> placeholder.color;
     property enabled <=> input.enabled;
-    property has-focus <=> input.has-focus;
+    property <bool> has-focus: input.has-focus;
```

Comment:
Ah, I guess this will be “fixed” once we use the new syntax for these?

Reply:
yes